CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- [...]
+
+# v2.0.2 (12/04/2019)
+
 - **[UPDATE]** Added aria attributes and focus trap on Modals
```

Comment:
nit: ideally, instead of "Modals"  I'd prefer: `<Modal>`,  `<ConfirmationModal>` and  `<SuccessModal>`